base/base_tests/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(
   buffer_vector_test.cpp
   bwt_tests.cpp
   cache_test.cpp
+  clustering_map_tests.cpp
```

Comment:
..._tests or test? In out project test files have ending "_test.cpp" and "_tests.cpp". Let's agreed about the naming. What do you think about it? Af for me it most cases I prefer _tests.

Reply:
I prefer "tests", because it's really a pain to rename file when number of tests becomes greater than one.

Reply:
I agree.

Reply:
The tradition in the project seems to be "_test", but I agree that "_tests" is better, so let's break this tradition (as we already have elsewhere, anyway).